docs/manual/scala/guide/broker/code/docs/scaladsl/mb/AnotherServiceImpl.scala

```diff
@@ -51,5 +53,51 @@ class AnotherServiceImpl(helloService: HelloService) extends AnotherService {
     //#subscribe-to-topic-with-metadata
 
   }
+
+  def skipMessages = {
+    //#subscribe-to-topic-skip-messages
+    helloService
+      .greetingsTopic()
+      .subscribe
+      .atLeastOnce(
+        Flow[GreetingMessage].map {
+          case msg@GreetingMessage("Kia ora") => doSomethingWithTheMessage(msg)
+          case _ => Done // Skip all messages that do not start with "Hello"
+        }
+      )
+    //#subscribe-to-topic-skip-messages
+  }
+
+  def partitionMessages = {
+    //#subscribe-to-topic-partition-messages
+    helloService
+      .greetingsTopic()
+      .subscribe
+      .atLeastOnce(flow)
```

Comment:
I would add the same disclaimer here (link to akka docs for Graph dsl, "this is advanced", ...).

(see my comment above)